src/themes/teams/components/Menu/menuStyles.ts

```diff
@@ -42,4 +42,13 @@ export default {
       listStyleType: 'none',
     }
   },
+  divider: ({ props: { pointing, vertical, pills, underlined } }) => ({
+    ...(pointing &&
+      vertical && {
+        marginBottom: `${pxToRem(12)}`,
```

Comment:
have agreed with @mnajdova to prevent duplication of `pxToRem` styles between `MenuDivider` and `MenuItem` styles